child_compassion/models/child_compassion.py

```diff
@@ -323,6 +323,10 @@ class CompassionChild(models.Model):
                                    readonly=True)
     portrait = fields.Binary(related='pictures_ids.headshot')
     fullshot = fields.Binary(related='pictures_ids.fullshot')
+    child_disaster_impact_ids = fields.One2many(
+        'child.disaster.impact', 'fo_disaster_alert_id', 'Child Disaster '

```

Comment:
Instead of `'fo_disaster_alert_id'` you should put `'child_id'` (it is the inverse name field on the related model)